util/intensity_explorer.py

```diff
@@ -0,0 +1,623 @@
+
+    fig.savefig(self.outfile + '_z_vs_I', transparent=True)
+    plt.close()
+
+
+def data_from_unmerged_mtz(filename):
+  """
+  Produce a minimal reflection table from an MTZ file.
+
+  The returned reflection table will not contain all the standard
+  columns, only those that are necessary for the IntensityDist class.
+
+  :param filename: Name of an unmerged MTZ input file.
+  :type filename: str
+  :return: A reflection table object, containing only the columns
+    * ``miller_index``
+    * ``intensity.sum.value``
+    * ``intensity.sum.variance``
+    * ``xyzobs.px.value``
+    * ``id``
+  :rtype: dials.array_family_flex_ext.reflection_table
+  """
+
+  from iotbx import mtz
+  from dxtbx.model import Crystal, Experiment, ExperimentList
+
+  m = mtz.object(filename).crystals() # Parse MTZ, with lots of useful methods.
+
+  # Get some data and turn it into a reflection table and experiment list.
+  #First, the reflection table
+  cols = m[0].datasets()[0].columns()  # Gets column objects.
+  col_dict = {c.label(): c for c in cols}  # A dict of all the columns.
+  h, k, l = (
+    col_dict[label].extract_values().as_double().iround()
+    for label in ('H', 'K', 'L')
+  )
+  intensity, sigI, x, y = (
+    col_dict[label].extract_values().as_double()
+    for label in ('I', 'SIGI', 'XDET', 'YDET')
+  )
+  # Honestly flex?!  Oh well, for now, we have to go round the houses:
+  frame = col_dict['BATCH'].extract_values().as_double().iround().as_double()
+
+  rtable = flex.reflection_table()
+  rtable['miller_index'] = flex.miller_index(h, k, l)
+  rtable['intensity.sum.value'] = intensity
+  rtable['intensity.sum.variance'] = flex.pow2(sigI)
+  rtable['xyzobs.px.value'] = flex.vec3_double(x, y, frame)
+  rtable['id'] = flex.int(rtable.size(), 0)
+
+  # Now generate a corresponding experiment list.
+  id = flex.vec3_double([(1, 0, 0),
+                         (0, 1, 0),
+                         (0, 0, 1)])
+  # Each Crystal object needs to be constructed from xyz unit cell
+  # parameters and a space group.
+  abc = [ m[0].unit_cell().orthogonalize(vec) for vec in id ]
+  space_group = m[0].crystal_symmetry().space_group()
+  crystal_params = abc + [space_group]
+
+  elist = ExperimentList([Experiment(crystal=Crystal(*crystal_params))])
+
+  return rtable, elist
+
+
+def data_from_pickle_and_json():
+    from dials.util.options import OptionParser, flatten_reflections,\
+      flatten_experiments
+    import libtbx.load_env
+
+    help_message = '''
+
+  Generates a dials.array_family.flex.reflection_table and a 
+  dxtbx.model.experiment_list.ExperimentList from a reflections.pickle and an
+  experiments.json.
+
+  Examples:
+
+    dials.util.intensity_explorer indexed.pickle experiments.json
+
+    dials.util.intensity_explorer integrated.pickle integrated_experiments.json
+
+    '''
+
+    # Create the parser
+    usage = ("usage: %s [options] reflections.pickle experiments.json" %
+             libtbx.env.dispatcher_name)
```

Comment:
You spelt out the name in the comment above, so can do it here as well. Removes the libtbx dependency.